src/assert.sh

```diff
@@ -318,8 +318,9 @@ function assert_greater_or_equal_than() {
 
 function assert_line_count() {
```

Comment:
I feel sorry for opening this can of worms :-).

thanks for working thru all these problem.

Reply:
You opened a pandora box... nah, it is also fun - even though I really dislike bash 😝

Reply:
"you just need to count the lines" - they said